### PR TITLE
script cleanup + new script for 2 virtual networks

### PR DESCRIPTION
**Pull request by Mopster**
Cleaned up and formatted the original 03-setup-openvnet.sh script for readability.
Added a second script (03-setup-openvnet.sh) which will create a second virtual network (10.0.2.0/24) using the lxc2's.

### CODE REVIEW

File: kvm-guests/192.168.3.10-vnetdev/vnetscript/03-setup-openvnet-2vn.sh [hunk 1]
```diff
@@ -0,0 +1,164 @@
+!/bin/bash
```
Comment (hansode) [inline kvm-guests/192.168.3.10-vnetdev/vnetscript/03-setup-openvnet-2vn.sh]: `!#/bin/bash`

Comment (hansode) [inline kvm-guests/192.168.3.10-vnetdev/vnetscript/03-setup-openvnet-2vn.sh]: `#!/bin/bash`

Comment (Mopster) [inline kvm-guests/192.168.3.10-vnetdev/vnetscript/03-setup-openvnet-2vn.sh]: Corrected

**Comment by hansode** [comment]
LGTM :+1:

**Pull request merged**